idseq_dag/steps/run_star.py

```diff
@@ -9,6 +10,8 @@
 import idseq_dag.util.count as count
 import idseq_dag.util.validate_constants as vc
 
+RE_SPLIT = re.compile('[/\t]')
```

Comment:
I feel like this might be too wide. Can we make the change more minimal by only accepting '/1\t' and '/2\t' in addition to '\t', to fix just the particular case we're looking at?

Reply:
Yes, I'll do that, but I'm curious if there isn't a document where I can find all standards, and craft the correct validation regexp.

Reply:
Here's documentation of the FASTQ format: https://en.wikipedia.org/wiki/FASTQ_format
It doesn't say much about the format of the read id.
Here's documentation of the specific read id format that comes out of Illumina sequencers, which is implicitly what we're assuming we have most of the time: https://help.basespace.illumina.com/articles/descriptive/fastq-files/